Bio/SeqIO/_index.py

```diff
@@ -39,6 +39,8 @@
 
 
 class SeqFileRandomAccess(_IndexedSeqFileProxy):
+    """Random Access File generic."""
```

Comment:
How about something more like this?:

``Base class for defining random access to sequence files.``